webodf/lib/ops/OdtDocument.js

```diff
@@ -938,6 +979,48 @@ ops.OdtDocument = function OdtDocument(odfCanvas) {
     };
 
     /**
+     * Process steps being inserted into the document. Will emit a steps inserted signal on
+     * behalf of the caller
+     * @param {!{position: !number}} args
+     * @return {undefined}
+     */
+    this.handleStepsInserted = function(args) {
+        stepsTranslator.handleStepsInserted(args);
+        // signal not used in webodf, but 3rd-party (NVivo)
+        eventNotifier.emit(ops.OdtDocument.signalStepsInserted, args);
+    };
+
+    /**
+     * Process steps being removed from the document. Will emit a steps removed signal on
+     * behalf of the caller
+     * @param {!{position: !number}} args
+     * @return {undefined}
+     */
+    this.handleStepsRemoved = function(args) {
+        stepsTranslator.handleStepsRemoved(args);
+        // signal not used in webodf, but 3rd-party (NVivo)
+        eventNotifier.emit(ops.OdtDocument.signalStepsRemoved, args);
```

Comment:
These events will occur out-of-order if not added to the operation's own event list.